tests/app/shared/issue/description/description.component.spec.ts

```diff
@@ -91,4 +101,34 @@ describe('DescriptionComponent', () => {
     expect(issueUpdatedEmit).toHaveBeenCalledTimes(1);
     expect(resetCall).toHaveBeenCalledTimes(1);
   });
+
+  it('should reset updates if description edit is cancelled', () => {
+    const issueUpdatedEmit = spyOn(descriptionComponent.issueUpdated, 'emit');
+    const resetCall = spyOn(descriptionComponent, 'resetToDefault');
+
+    descriptionComponent.ngOnInit();
+    descriptionComponent.changeToEditMode();
+
+    issueService.getIssue.and.callFake((x: number) => of(thisIssue));
+    issueService.updateIssue.and.callFake((x: Issue) => of(x));
+    descriptionComponent.cancelEditMode();
+
+    expect(issueUpdatedEmit).toHaveBeenCalledTimes(1);
+    expect(resetCall).toHaveBeenCalledTimes(1);
+  });
+
+  it('should cancel editing only if confirmed on confirmation dialog', () => {
```

Comment:
```suggestion
  it('should cancel edit mode only if confirmed in confirmation dialog', () => {
```
Might be better to use consistent naming for the action and function